crates/mpc-protocols/src/mpc_events.rs

```diff
@@ -1,13 +1,35 @@
+use move_core_types::ident_str;
+use move_core_types::identifier::IdentStr;
+use move_core_types::language_storage::StructTag;
 use pera_types::{
     base_types::{ObjectID, PeraAddress},
     id::ID,
+    PERA_SYSTEM_ADDRESS,
 };
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
 /// Rust version of the Move [`pera_system::dwallet::CreatedProofMPCSessionEvent`] type.
 #[derive(Debug, Serialize, Deserialize, Clone, JsonSchema, Eq, PartialEq)]
-pub struct CreatedProofMPCSessionEvent {
+pub struct CreatedProofMPCEvent {
     pub session_id: ID,
     pub sender: PeraAddress,
 }
+
+pub const PROOF_MODULE_NAME: &IdentStr = ident_str!("proof");
+pub const CREATED_PROOF_STRUCT_NAME: &IdentStr = ident_str!("CreatedProofMPCSessionEvent");
```

Comment:
Could you please explain what the upside to making these `const &IdentStr` is? Just curious

Reply:
We don't know why, this is just the type being used in `StructTag`.